projects/packages/woocommerce-analytics/src/class-checkout-flow.php

```diff
@@ -141,6 +141,11 @@ public function capture_cart_view() {
 	 */
 	public function capture_checkout_view() {
 		global $post;
+
+		if ( ! $post ) {
```

Comment:
What do you think about being more explicit as to what we expect?

```suggestion
		if ( ! $post instanceof WP_Post ) {
```